### PR TITLE
added docker submit_system

### PR DESCRIPTION
**Pull request by v1kko**
Method to submit via docker, needs an update from jetto-tools as well

Depends on: https://gitlab.com/jintrac/jetto-pythontools/-/merge_requests/5

### CODE REVIEW

File: src/duqtools/jetto/_system.py [hunk 1]
```diff
@@ -85,13 +88,32 @@ def submit_slurm(job: Job):
...
             f.write(ret.stdout)
 
+    @staticmethod
+    def submit_docker(job: Job):
+
+        jetto_template = template.from_directory(job.dir)
+        jetto_config = config.RunConfig(jetto_template)
+        jetto_manager = jetto_job.JobManager()
+        extra_volumes = {
+            job.dir / '..' / 'imasdb': {
```
Comment (stefsmeets) [inline src/duqtools/jetto/_system.py:98]:
```suggestion
            job.dir.parent / 'imasdb': {
```

**Comment by v1kko** [comment]
Converted to draft, as we really need IMAS enabled jintrac container for this to work

**Comment by stefsmeets** [comment]
Depends on #484

**Review by stefsmeets — APPROVED** [review]
Looks good to me after you remove all the changes to the template! 🚀

File: src/duqtools/jetto/_system.py [hunk 2]
```diff
@@ -155,9 +181,13 @@ def copy_from_template(source_drc: Path, target_drc: Path):
...
-        jetto_extra = []
-        if (source_drc / 'jetto.ex').exists():
-            jetto_extra = [str(source_drc / 'jetto.ex')]
+        all_files = os.listdir(source_drc)
+
+        jetto_extra: List[str] = []
+        for regex in _EXTRA_FILE_REGEXES:
+            jetto_extra = jetto_extra + list(filter(regex.match, all_files))
+
+        jetto_extra = [str(source_drc / file) for file in jetto_extra]
```
[the remark this answers is not in the document]
Comment (v1kko) [reply src/duqtools/jetto/_system.py:190]: > `_EXTRA_FILE_REGEXES`, is that a filter for all the files that should be in the directory to start a jetto run?

Yes

**Pull request merged**